test/tree_sitter/node_test.rb

```diff
@@ -339,6 +339,12 @@ def mul(a, b)
   end
 end
 
+describe 'Enumerable' do
+  it 'should be' do
```

Comment:
Oh, a Yoda test.
Interesting it is.

Reply:
Waw I was wondering if you'd notice it :)